x-pack/plugins/infra/server/services/rules/rule_data_client.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { once } from 'lodash';
+import { CoreSetup, Logger } from 'src/core/server';
+import { TECHNICAL_COMPONENT_TEMPLATE_NAME } from '../../../../rule_registry/common/assets';
+import { RuleRegistryPluginSetupContract } from '../../../../rule_registry/server';
+import { logThresholdRuleDataNamespace } from '../../../common/alerting/logs/log_threshold';
+import { RuleRegistrationContext, RulesServiceStartDeps } from './types';
+
+export const createRuleDataClient = ({
+  registrationContext,
+  getStartServices,
+  logger,
+  ruleDataService,
+}: {
+  registrationContext: RuleRegistrationContext;
+  getStartServices: CoreSetup<RulesServiceStartDeps>['getStartServices'];
+  logger: Logger;
+  ruleDataService: RuleRegistryPluginSetupContract['ruleDataService'];
+}) => {
+  const initializeRuleDataTemplates = once(async () => {
+    const componentTemplateName = ruleDataService.getFullAssetName(
+      `${registrationContext}-mappings`
```

Comment:
@weltenwort apm uses `apm-mappings` https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/plugin.ts#L136, where as we use `observability.logs-mappings` and `observability.metrics-mappings`. Do I miss something here? Shouldn't we be consistent across different solutions?

Reply:
As discussed via another channel, I agree that it should be consistent and that APM should probably  use `observability.apm-mappings`.